frontend/src/v4/routes/components/invitationsDialog/invitationsDialog.component.tsx

```diff
@@ -89,24 +92,26 @@ export const InvitationsDialog = (props: IProps) => {
 		<Container className={props.className}>
 			{renderInvitationsList(!!props.invitations.length)}
 			{renderNoInvitationsInfo(!props.invitations.length)}
-			<Footer>
-				<CancelButton
-					type="button"
-					color="primary"
-					variant="text"
-					onClick={props.handleClose}
-				>
-					Cancel
-				</CancelButton>
-				<Button
-					type="button"
-					variant="contained"
-					color="secondary"
-					onClick={handleInvitationClick({})}
-				>
-					Add
-				</Button>
-			</Footer>
+			{!props.usersProvisionedEnabled && (
+				<Footer>
+					<CancelButton
+						type="button"
+						color="primary"
+						variant="text"
+						onClick={props.handleClose}
+					>
+						Cancel
+					</CancelButton>
+						<Button
+							type="button"
+							variant="contained"
+							color="secondary"
+							onClick={handleInvitationClick({})}
+						>
+							Add
+						</Button>
+				</Footer>
+			)}
```

Comment:
Users shouldn't be able to assign anyone as admin if disablePermissionsOnUI is true
![image](https://github.com/user-attachments/assets/8ca193e1-8450-41f0-9659-0dc6e1a4bb6a)